### PR TITLE
chore(docs): Add paths to test schema files

### PR DESCRIPTION
**Pull request by alexolivier**
Adds the URLs for the tests schema files for editor support.

### CODE REVIEW

File: docs/modules/policies/pages/authoring_tips.adoc [hunk 1]
```diff
@@ -46,7 +56,8 @@ YAML language server also supports per-directory settings. If all your Cerbos po
 [source,yaml,subs="+attributes"]
 ----
 yaml.schemas: {
-    "{latest-schema-url}": "/cerbos/*"
+    "{latest-policy-schema-url}": "/cerbos/*",
+    "{latest-test-schema-url}": "/cerbos/tests/*",
```
Comment (charithe) [inline docs/modules/policies/pages/authoring_tips.adoc:60]:
```suggestion
    "{latest-test-schema-url}": "/cerbos/**/*_test.yaml",
```

File: docs/modules/policies/pages/authoring_tips.adoc [hunk 2]
```diff
@@ -75,7 +86,8 @@ use({
             settings = {
                 yaml = {
                     schemas = {
-                        "{latest-schema-url}": "/cerbos/*"
+                        "{latest-policy-schema-url}": "/cerbos/*",
+                        "{latest-test-schema-url}": "/cerbos/tests/*"
```
Comment (charithe) [inline docs/modules/policies/pages/authoring_tips.adoc:90]:
```suggestion
                        "{latest-test-schema-url}": "/cerbos/**/*_test.yaml"
```

File: docs/modules/policies/pages/authoring_tips.adoc [hunk 3]
```diff
@@ -114,7 +126,8 @@ After the extension is installed, hit kbd:[Ctrl+,] or menu:File[Preferences > Se
 ----
 {
   "yaml.schemas": {
-    "{latest-schema-url}": "cerbos/*"
+    "{latest-policy-schema-url}": "cerbos/*",
+    "{latest-test-schema-url}": "/cerbos/tests/*",
```
Comment (charithe) [inline docs/modules/policies/pages/authoring_tips.adoc:130]:
```suggestion
    "{latest-test-schema-url}": "/cerbos/**/*_test.yaml",
```

File: docs/modules/policies/pages/authoring_tips.adoc [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
+:current-test-schema-url: {app-schema-url-current}{test-schema-file}
+:latest-test-schema-url: {app-schema-url-latest}{test-schema-file}
 
 = Policy authoring 
 
 == Tips for working with policies
 
 * Policies can be in either YAML or JSON formats. Accepted file extensions are `.yml`, `.yaml` or `.json`. All other extensions are ignored.
-* The JSON schema for Cerbos policies is available at `{current-schema-url}`. If you prefer to always use the latest version, it can be accessed at `{latest-schema-url}` as well. 
+* The JSON schema for Cerbos policies is available at `{current-policy-schema-url}` and xref:compile.adoc[tests] at `{current-test-schema-url}`. If you prefer to always use the latest version, it can be accessed at `{latest-policy-schema-url}` and `{latst-test-schema-url}`.
```
Comment (charithe) [inline docs/modules/policies/pages/authoring_tips.adoc:15]:
```suggestion
* The JSON schema for Cerbos policies is available at `{current-policy-schema-url}` and the xref:compile.adoc[test suite] schema is available at `{current-test-schema-url}`. If you prefer to always use the latest version, they can be accessed at `{latest-policy-schema-url}` and `{latst-test-schema-url}` respectively. 
```

Comment (oguzhand95) [reply docs/modules/policies/pages/authoring_tips.adoc:15]: `{latst-test-schema-url}` -> ``{latest-test-schema-url}``

**Review by charithe — APPROVED** [review]

**Pull request merged**